cmd/ctr/commands/run/run_unix.go

```diff
@@ -97,6 +97,15 @@ func NewContainer(ctx context.Context, client *containerd.Client, cliContext *cl
 		id = cliContext.Args().Get(1)
 	}
 
+	platform := cliContext.String("platform")
+	if platform == "" {
+		plat := platforms.DefaultSpec()
+		// Use linux as the default container platform, other Unix platforms
+		// do not currently generate valid specs or have working runtimes
```

Comment:
FreeBSD has a (draft) spec and working runtimes, though.

Reply:
Right now in runj I'm ignoring and compensating for the Linux values here. I think even for running a Linux image I'd want to have a FreeBSD config; inside the jail FreeBSD's Linuxulator can run Linux binaries but the jail still needs to be specified correctly for the FreeBSD kernel.

Reply:
This could be different if there were a VM-based runtime with a Linux guest on FreeBSD (through bhyve, for example), but we don't have such a runtime today.

Reply:
Functionally this won't make a difference anyway since this value is just fed into https://github.com/containerd/containerd/blob/main/pkg/oci/spec.go#L95 , which does not yet have a freebsd implementation. My assumption is getting it to work on freebsd requires providing a complete runtime spec rather than rely on the spec generation. I'll update this section to limit the switch so it isn't just a confusing no-op on freebsd.